lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -1437,15 +1438,14 @@ prop_balanceTransactionValid
                 counterexample ("balanceTransaction failed: " <> show err) False
   where
     prop_expectFeeExcessSmallerThan
-        :: CardanoApi.Lovelace
-        -> CardanoApi.Tx (CardanoApiEra era)
+        :: Coin
+        -> Tx era
         -> UTxO era
         -> Property
     prop_expectFeeExcessSmallerThan lim tx utxo = do
         let fee = txFee tx
         let minfee = minFee tx utxo
-        let unLovelace (CardanoApi.Lovelace x) = x
-        let delta = CardanoApi.Lovelace $ (unLovelace fee) - (unLovelace minfee)
+        let delta = fee <-> minfee
```

Comment:
I think we could also consider using [`~~`](https://hackage.haskell.org/package/groups-0.5.3/docs/Data-Group.html#v:-126--126-) from [`Data.Group`](https://hackage.haskell.org/package/groups-0.5.3/docs/Data-Group.html#t:Group) in preference to `<->` here, as:

- [`MaryValue`](https://input-output-hk.github.io/cardano-ledger/cardano-ledger-mary/Cardano-Ledger-Mary-Value.html#t:MaryValue) and [`Coin`](https://input-output-hk.github.io/cardano-ledger/cardano-ledger-core/Cardano-Ledger-Coin.html#t:Coin) are already both instances of [`Data.Group`](https://hackage.haskell.org/package/groups-0.5.3/docs/Data-Group.html#t:Group).
- [`Data.Group`](https://hackage.haskell.org/package/groups-0.5.3/docs/Data-Group.html#t:Group) is arguably already fairly standard within the Hackage ecosystem.

What do you think?

Reply:
As discussed — I'd lean towards using `<->` for the sake of being similar to the ledger implementation. At least for now.